### PR TITLE
feat(documentation-v7): migrate the blockquote component stories

### PR DESCRIPTION
**Pull request by b1aserlu**

### CODE REVIEW

File: packages/documentation-v7/src/stories/components/blockquote/blockquote.snapshot.stories.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          language: bqMeta.argTypes.language.options,
+          caption: [false, true],
+        })
+          .map(args => ({ ...args, text: `${args.language.toUpperCase()}: ${args.text}` }))
+          .map(args => Default({ ...bqMeta.args, ...args }))}
+      </div>
+    );
+  })}
+</div>
+*/
```
Comment (oliverschuerch) [inline packages/documentation-v7/src/stories/components/blockquote/blockquote.snapshot.stories.ts:74]: Remove!

File: packages/documentation-v7/src/stories/components/blockquote/blockquote.stories.ts
Comment (oliverschuerch) [inline packages/documentation-v7/src/stories/components/blockquote/blockquote.stories.ts:1]: File format looks broken.
Fix it!

File: packages/documentation-v7/src/stories/components/blockquote/blockquote.stories.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    const result= args.caption
+    ? html`<figure>
+    ${blockquote}
+    <figcaption class="blockquote-footer">
+      ${args.captionAuthor} <cite title=${args.sroucetitle}>${args.captionSource}</cite>
+    </figcaption>
+    </figure>`
+    : blockquote
+
+  return result;
```
Comment (oliverschuerch) [inline packages/documentation-v7/src/stories/components/blockquote/blockquote.stories.ts:105]: You do not need to create a const here, simply return the value:
```javascript
return args.caption ? ....
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/documentation-v7/src/stories/components/blockquote/blockquote.snapshot.stories.ts (hunk 1 above)
Comment (b1aserlu) [reply packages/documentation-v7/src/stories/components/blockquote/blockquote.snapshot.stories.ts]: removed

File: packages/documentation-v7/src/stories/components/blockquote/blockquote.stories.ts
Comment (b1aserlu) [reply packages/documentation-v7/src/stories/components/blockquote/blockquote.stories.ts:1]: reformated

File: packages/documentation-v7/src/stories/components/blockquote/blockquote.stories.ts (hunk 2 above)
Comment (b1aserlu) [reply packages/documentation-v7/src/stories/components/blockquote/blockquote.stories.ts]: implemented as suggested

File: packages/documentation-v7/src/stories/components/blockquote/blockquote.docs.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { Meta, Canvas, Controls } from '@storybook/blocks';
+import * as BlockquoteStories from './blockquote.stories.ts';
+
+<Meta of={BlockquoteStories} />
+
+# Blockquote
+
+<div className="lead">
+  For quoting blocks of content from another source within your document. Wrap `&lt;blockquote
+  class="blockquote"&gt;` around any HTML as the quote.
```
Comment (oliverschuerch) [inline packages/documentation-v7/src/stories/components/blockquote/blockquote.docs.mdx:10]: This works not as I expected. Try this one and if this does also not work, implement the first solution without any ` around the blockquote.
```suggestion
  For quoting blocks of content from another source within your document. Wrap `<blockquote
  class="blockquote">` around any HTML as the quote.
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/documentation-v7/src/stories/components/blockquote/blockquote.docs.mdx (hunk 3 above)
Comment (b1aserlu) [reply packages/documentation-v7/src/stories/components/blockquote/blockquote.docs.mdx]: Sorry, I resolved the last one, but forgot to actually do it. It is fixed now.

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**